addons/website_event/views/website_visitor_views.xml

```diff
@@ -32,6 +32,7 @@
                 <button name="%(website_event.event_registration_action_from_visitor)d"
                     type="action"
                     class="oe_stat_button" icon="fa-ticket"
+                    groups="event.group_event_user"
                     attrs="{'invisible': [('event_registration_count', '=', 0)]}">
                     <field name="event_registration_count" widget="statinfo" string="Registrations"/>
                 </button>
```

Comment:
I can't click on any visitor (even without registration), could you have a look?

Reply:
There was a similar issue on `website_event_track`.